cloud/google/machineactuator.go

```diff
@@ -399,15 +410,13 @@ func (gce *GCEClient) PostCreate(cluster *clusterv1.Cluster, machines []*cluster
 		return fmt.Errorf("error creating service account for ingress controller: %v", err)
 	}
 
-	if len(machines) > 0 {
-		config, err := gce.providerconfig(machines[0].Spec.ProviderConfig)
-		if err != nil {
-			return fmt.Errorf("error creating ingress controller: %v", err)
-		}
-		err = CreateIngressController(config.Project, cluster.Name)
-		if err != nil {
-			return fmt.Errorf("error creating ingress controller: %v", err)
-		}
+	clusterConfig, err := gce.clusterproviderconfig(cluster.Spec.ProviderConfig)
+	if err != nil {
+		return fmt.Errorf("error creating ingress controller: %v", err)
```

Comment:
Would like error message to provide more context on what aspect ingress controller failed. Otherwise debugging whether it was this line or the line a few lines later will be harder.